### PR TITLE
New Feature: Add an Issue `in-develop` labeler and functions to close these issues 

### PR DESCRIPTION
**Pull request by Thomas-Boi**
# **New feature section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *New Feature: a brief description of feature*

## This PR adds/fixes

-  A new workflow, `in_develop_labeler` or `label-bot`:
    - It will parse the PR's main message, locate the issues in them, and label those issues `in-develop`
    - The workflow only run when the PR is successfully merged.
    - Issues are located by checking the line starting with `**This PR closes`. If there are no issues located, the workflow does nothing.
- An extra step for `build-bot`:
  - When the bot is run, it will locate all issues with `in-develop` label and close them.

## Notes

Both workflows work when merged into `develop`. However, the `**This PR closes` phrase belongs to the Template Update PR, which requires merge to `master` for it work.

EDIT: The features here were tested in the Thomas-Boi/devicon repo. It works as intended and you can view the result by looking at the closed Issues.

### CODE REVIEW

File: .github/scripts/icomoon_build.py [hunk 1]
```diff
@@ -49,6 +49,11 @@ def main():
         print("Creating the release message by querying the GitHub API...")
         get_release_message(args.token)
 
+        print("Closing the issues with the label of `in-develop`.")
```
Comment (Panquesito7) [inline .github/scripts/icomoon_build.py:52]:
```suggestion
        print("Closing issues with the `in-develop` label.")
```

File: .github/scripts/build_assets/api_handler.py [hunk 2]
```diff
@@ -1,8 +1,14 @@
 import requests
 import sys
 import re
+from typing import List
 
 
+# our base url which leads to devicon
+base_url = "https://api.github.com/repos/devicons/devicon/"
+# testing url
+# base_url = "https://api.github.com/repos/Thomas-Boi/devicon/"
```
Comment (amacado) [inline .github/scripts/build_assets/api_handler.py:10]: You might want to remove this comment?

Comment (Thomas-Boi) [reply .github/scripts/build_assets/api_handler.py]: Thanks for spotting that

**Review by amacado — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Comment by Thomas-Boi** [comment]
Thanks for the reviews guys 😄. I really want to get this one for the next release

**Pull request merged**